models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -24,10 +24,10 @@
     "references": [],
     "network_data_format": {
```

Comment:
need a specific schema that has two networks as discussed offline

Reply:
Done! Thank you!